include/ibamr/IBFESurfaceMethod.h

```diff
@@ -547,7 +612,10 @@ class IBFESurfaceMethod : public IBStrategy
     IBTK::FEDataManager::SpreadSpec d_default_spread_spec;
     std::vector<IBTK::FEDataManager::InterpSpec> d_interp_spec;
     std::vector<IBTK::FEDataManager::SpreadSpec> d_spread_spec;
-    bool d_use_jump_conditions = false;
+    bool d_use_pressure_jump_conditions, d_use_velocity_jump_conditions;
+    bool d_use_l2_lagrange_family;
+    bool d_compute_fluid_traction;
+    bool d_traction_interior_side;
```

Comment:
Do these not get default values?

Reply:
I thought we fixed this yesterday.

Reply:
Currently we assign some of the default values in `commonConstructor` and some in `IBFESurfaceMethod.h`. Do you suggest to have them all in the .h file? Is there any consistent way to set these values?

Reply:
You should set all of the initial values that are consistent between all configurations in the header and get rid of the initialization in the `.cpp` file.

Reply:
(This makes it easier to see what the defaults are.)

Reply:
Also, setup the defaults for IB, not IIM. We don't want to change the default behavior at this point.